### ISSUE TITLE
protobuf codegen needs to rewrite some reserved keywords

### ISSUE THREAD

**Issue by stevej** [linkerd/linkerd#921]
While attempting to implement the grpc interop service, I ran into a small issue

```
// A block of data, to simply increase gRPC message size.
message Payload {
  // DEPRECATED, don't use. To be removed shortly.
  // The type of data in body.
  PayloadType type = 1;
  // Primary contents of payload.
  bytes body = 2;
}
```
[error] /Users/stevej/src/2/linkerd/grpc/interop/target/scala-2.11/src_managed/main/compiled_protobuf/grpc/testing/messages.pb.scala:85: identifier expected but 'type' found.
[error]   type: Option[grpc.testing.PayloadType.Value],
[error]   ^```

`type` needs to be rewritten to _type (or something else).

**Comment by olix0r** [comment]
Or we can just backtick-escape these definitions like:

```scala
`type`: Option[grpc.testing.PayloadType.Value
```

**Comment by stevej** [comment]
This was fixed in #977
